dm/ui/plugins/vite-plugin-next-react-router/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "vite-plugin-next-react-router",
+  "version": "0.6.2",
+  "main": "index.js",
+  "module": "./index.mjs",
+  "exports": {
+    ".": {
+      "require": "./index.js",
+      "import": "./index.mjs",
+      "types": "./index.d.ts"
+    }
+  },
+  "dependencies": {
+    "consola": "^2.15.3",
+    "fast-glob": "^3.2.7",
+    "lodash": "^4.17.21"
+  },
+  "peerDependencies": {
+    "react-router-dom": ">=6",
+    "vite": ">=2"
+  }
```

Comment:
should we use persistent patch versions?

Reply:
it's fine here, peerDependencies means it should work with another package and it is compatible with this version range, like a typical plugin system